src/target/stm32h7.c

```diff
@@ -343,25 +371,55 @@ static bool stm32h7_flash_unlock(target_s *const target, const uint32_t addr)
 	return !(target_mem_read32(target, regbase + FLASH_CR) & FLASH_CR_LOCK);
 }
 
+/* Helper for offsetting FLASH_CR bits correctly */
+static uint32_t stm32h7_flash_cr(uint32_t sector_size, const uint32_t ctrl, int snb)
+{
+	uint32_t command = ctrl;
+	/* H74x, H72x IP: 128 KiB and has PSIZE */
+	if (sector_size == FLASH_SECTOR_SIZE) {
+		command = ctrl | (snb * FLASH_CR_SNB_1);
```

Comment:
Given `command` is already a copy of `ctrl` at this point, do you see lower Flash usage or any change if you use `|=` instead of `= ctrl |`?

Reply:
We'd prefer the `|=` variant then, it's clearer what the intended meaning is and that `ctrl`'s value is already in `command` intentionally. Compiler looks to have done the pivot for us anyway, which is nice, but we'd like to not rely on that.